jsapp/js/account/usage/usageProjectBreakdown.module.scss

```diff
@@ -1,7 +1,96 @@
 @use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
 
 .root {
   width: auto;
-  padding-top: sizes.$x40;
+  padding-bottom: sizes.$x40;
+  padding-top: sizes.$x20;
   margin: 0 sizes.$x40;
+  height: 100%;
+  overflow-x: auto;
+}
+
+table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.projects {
+  padding-right: 15%;
+  padding-left: 0%;
+  white-space: nowrap;
+}
+
+.wrap {
+  width: sizes.$x120;
+}
+
+th {
+  font-weight: 700;
+  font-size: sizes.$x14;
+  color: colors.$kobo-gray-40;
+  text-align: left;
+  padding: 1.5%;
```

Comment:
The horizontal padding could get removed here to make these column headers line up better (maybe `padding-block: 1.5%`?):

![image](https://github.com/kobotoolbox/kpi/assets/7819986/f3d67ca7-07e2-40ba-8c20-ebff40279e85)